test/e2e/dra/latency.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	resourceapi "k8s.io/api/resource/v1alpha3"
+	"k8s.io/kubernetes/pkg/kubelet/metrics"
+	"k8s.io/kubernetes/test/e2e/feature"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2emetrics "k8s.io/kubernetes/test/e2e/framework/metrics"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+const (
+	PrepareResourcesDurationSum   = metrics.DRAPrepareResourcesDurationKey + "_sum"
+	UnprepareResourcesDurationSum = metrics.DRAUnprepareResourcesDurationKey + "_sum"
+)
+
+var _ = framework.SIGDescribe("node")("DRA latency metrics", framework.WithSerial(), feature.DynamicResourceAllocation, func() {
+	f := framework.NewDefaultFramework("dra")
+
+	// The driver containers have to run with sufficient privileges to
+	// modify /var/lib/kubelet/plugins.
+	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
+
+	ginkgo.Context("DRA latency", func() {
```

Comment:
Can this be moved to `e2e_node`?

The less E2E tests we need to run serial the better.